framework/src/application/system_dirs.ts

```diff
@@ -13,8 +13,8 @@
  */
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types,@typescript-eslint/explicit-function-return-type
-export const systemDirs = (appLabel: string, tempPath: string) => ({
-	temp: `${tempPath}/${appLabel}/`,
-	sockets: `${tempPath}/${appLabel}/sockets`,
-	pids: `${tempPath}/${appLabel}/pids`,
+export const systemDirs = (appLabel: string, rootPath: string) => ({
+	temp: `${rootPath}/${appLabel}/`,
```

Comment:
this needs to be updated. this path is no longer `temp`
maybe like
```
{
  root: `${rootPath}/${appLabel}`,
  data: `${rootPath}/${appLabel}/data`,
  tmp: `${rootPath}/${appLabel}/tmp`,
  logs: `${rootPath}/${appLabel}/logs`,
 // maybe remove below
  sockets: `${rootPath}/${appLabel}/tmp/sockets`,
  pid: `${rootPath}/${appLabel}/tmp/pid`,
}
```
Also `tmp` is more common I think

Reply:
where will the sockets and pid go? who is using `root`?

Reply:
maybe `root` is not needed, `sockets` and `pid` should go under tmp